opennms-webapp/src/main/webapp/admin/notification/noticeWizard/choosePath.jsp

```diff
@@ -152,7 +187,7 @@
         </tr>
         <tr>
           <td width="10%" valign="top" align="left">
-            <label>Parameter:</label>
+            <label>Var-Bind:</label>
```

Comment:
I know you just renamed it to remove the clash between the other `parameter` we can set, but is `Var-Bind` a good name here? For what can it be used? 
I know that we can use it as some kind of filter. For example when you want to fetch only specific BSMs you can add this in your notification:

```
<varbind>
   <vbname>businessServiceName</vbname>
   <vbvalue>~^Virtual\ Host\ Cluster\ (FUL|FRA)</vbvalue>
</varbind>
```

Could make "Variable Filter" make more sense? Any other ideas?

@Bonrob2 This is definitely something we should add into the docs, since I don't think we have it already somewhere.

Reply:
I don't know. I would favor var-bind since it corresponds to the XML-structure.

Reply:
@christianpape Sounds legit.

Then we just should add that into the docs accordingly. The name isn't that important right now.

Reply:
I've created an issue for the documenation part: https://issues.opennms.org/browse/NMS-13273